lib/gc/Dialect/LLVMIR/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+add_mlir_dialect_library(MLIRGENDialect
```

Comment:
Should the directory name be "GEN" instead of "LLVMIR"?

Reply:
No, the dialect is at the same level as llvmir, this follows the upstream structure.